src/Core/Universe.m

```diff
@@ -1033,7 +1033,7 @@ - (void) carryPlayerOn:(StationEntity*)carrier inWormhole:(WormholeEntity*)wormh
 		[player setWormhole:wormhole];
 		[player addScannedWormhole:wormhole];
 
-		ShipScriptEventNoCx(player, "shipWillEnterWitchspace", OOJSSTR("carried"));
+		ShipScriptEventNoCx(player, "shipWillEnterWitchspace", OOJSSTR("carried"), INT_TO_JSVAL([wormhole destination]));
```

Comment:
Isn't this the same as the somewhat simpler
```
ShipScriptEventNoCx(player, "shipWillEnterWitchspace", OOJSSTR("carried"), INT_TO_JSVAL(dest));
```
dest is set as [wormhole destination] a few lines earlier.

Reply:
Yep, good call.